README.md

```diff
@@ -1,6 +1,23 @@
 # Welcome to Crowbar Angular Prototype
 [![Build Status](https://travis-ci.org/crowbar/crowbar-ui.svg?branch=master)](https://travis-ci.org/crowbar/crowbar-ui)
 
+### Setting up a development environment with Docker
+
+1. make sure `docker` is installed and the `docker` daemon is running.
+2. build the image from the [Dockerfile](Dockerfile):
+```
+docker build -t crowbar-ui .
+```
+3. run the container:
+```
+docker run -d --name "crowbar-ui_container" -v $(pwd)/assets:/crowbar-ui/assets -p 3000:3000 crowbar-ui
+```
+4. run the `gulp` watchdog:
+```
+docker exec -ti crowbar-ui_container gulp
+```
+5. work on the assets folder in your local git clone and the changes will be immediately visible in the browser.
```

Comment:
i didn't know that, probably due to the missing/outdated documentation on how to setup the dev env with all dependencies on the host.

can you please document what exactly is to be done when you change other files than the assets folder?

regarding the unit tests, can't we just run them inside the container? we have all necessary files there i believe.